cookies/giles.py

```diff
@@ -621,12 +683,32 @@ def process_details(data, upload_id, username):
         text_content_type = text_data.get('content-type')
         # text_uri = '%s/files/%s' % (giles, text_data.get('id'))
         text_uri = text_data.get('url')
-        _create_content_resource(resource, __text__, creator, text_uri,
-                                 text_uri, content_type=text_content_type)
+        content_resource = _create_content_resource(resource, __text__, creator,
+                                                    text_uri, text_uri,
+                                                    content_type=text_content_type)
+        Relation.objects.create(
+            source=content_resource,
+            predicate=__creator__,
+            target=Value.objects.create(name='PDF Extract'),
```

Comment:
This string should not be hard-coded here but use a constant that is then also used in PAGE_CREATOR_MAP.

Reply:
and actually those should probably be settings, so they are easy to change if we change out the ocr backend for example.